Packs/CommonScripts/Scripts/ReadPDFFileV2/ReadPDFFileV2.py

```diff
@@ -572,6 +610,8 @@ def extract_data_from_pdf(path: str, user_password: str, entry_id: str, max_imag
                                                  dec_file_path=f'{working_dir}/DecWorkingReadPDF.pdf',
                                                  encrypted=encrypted,
                                                  user_password=user_password)
+        # Get hash contexts
+        hash_contexts = extract_hash_contexts_from_pdf_file(cpy_file_path)
```

Comment:
Why running here and not on the text file like the URL and EMAIL? Lets keep the code clean and consistent.